routes/index.js

```diff
@@ -62,20 +62,20 @@ router.get('/api/create', (req, res) => {
   sql = 'INSERT INTO SEATS (seat_id, seat_number, envelopes_completed, envelopes_ready, is_taken, game_id, team_id) VALUES ?';
   db.query(sql, [values], function (err, result) {
     if (err) {
-      console.log(err);
+      console.log("ERROR: Unable to insert into seats" + err);
```

Comment:
you should have some kind of separator between the log message and the `err`. Either `"... seats: " + err` or you can pass multiple arguments into console.log like `"...seats", err` and it will separate them for you but the formatting is a little funky

Reply:
I just made those changes, I think it looks a lot better now.